ingest/ingest_pipeline.py

```diff
@@ -487,17 +502,27 @@ def extract_from_anndata(self):
         )
         if self.anndata.validate():
             self.report_validation("success")
+            # process matrix data
+            ### TODO: how to associate "raw_count" cells to anndata file
```

Comment:
Per convention, let's note the SCP Jira ticket this TODO corresponds to, e.g. `TODO (SCP-9876): how...`.

Reply:
Done `492d032`